chains.json

```diff
@@ -31,6 +31,7 @@
         "exponent": 6
       }
     ],
+    "seedNodes": [],
```

Comment:
looking at this diff: `testRpc` should be plural as well (`testRpcs`)

Reply:
Changed those